examples/aio-sandbox/aio-sandbox.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: agents.x-k8s.io/v1alpha1
+kind: Sandbox
+metadata:
+  name: aio-sandbox-example
+spec:
+  podTemplate:
+    metadata:
+      labels:
+        sandbox: aio-sandbox-example
+    spec:
+      containers:
+        - name: aio-sandbox
+          image: ghcr.io/agent-infra/sandbox:1.0.0.151
```

Comment:
use `latest`?

Reply:
latest `1.0.0.152`

Reply:
hI @ycjcl868 , I tried it but seems like not released yet in [ghcr](https://github.com/agent-infra/sandbox/pkgs/container/sandbox).